docs/references/ic-interface-spec.md

```diff
@@ -2273,7 +2273,25 @@ Indicates various information about the canister. It contains:
 
 -   A SHA256 hash of the module installed on the canister. This is `null` if the canister is empty.
 
--   The actual memory usage of the canister.
```

Comment:
I would keep this line and maybe make it a bit clearer that this refers to the total memory usage and additionally, create a new section that describes the detailed breakdown of memory types. As long as we keep both fields in the canister status result, both should be listed for consistency.